File: app/src/main/java/com/example/circleandsquareareas/MainActivity.java
package com.example.circleandsquareareas;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        int X = 3;
        double circleArea = Math.PI*X*X;
        System.out.println("Площадь круга = " + circleArea);

        double squareArea = X*X;
        System.out.println("Площадь квадрата = " + squareArea);

    }
}

File: app/src/main/java/com/example/circleandsquareareas/MainActivity.kt
package com.example.circleandsquareareas

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val X = 4
        val circleArea = Math.PI * X * X
        Log.d("DZ", "Площадь круга =$circleArea")
        val squareArea = (X * X).toDouble()
        Log.d("DZ", "Площадь квадрата =$squareArea")

        when {
            (circleArea > squareArea) -> {
                Log.d("DZ", "Площадь круга больше площади квадрата")
            }
            (circleArea < squareArea) -> {
                Log.d("DZ", "Площадь круга меньше площади квадрата")
            }
            else -> {
                Log.d("DZ", "Площадь круга равна площади квадрата")
            }
        }
    }
}